tests/test_svs_disp.py

```diff
@@ -0,0 +1,28 @@
+import subprocess
+import unittest
+
+from base import SD_VM_Local_Test
+
+
+class SD_SVS_Disp_Tests(SD_VM_Local_Test):
+    def setUp(self):
+        self.vm_name = "sd-svs-disp"
+        super(SD_SVS_Disp_Tests, self).setUp()
+
+    def test_sd_client_package_installed(self):
+        pkg = "securedrop-workstation-svs-disp"
+        self.assertTrue(self._package_is_installed(pkg))
+
+    def test_sd_svs_libreoffice_installed(self):
+        self.assertTrue(self._package_is_installed("libreoffice"))
+
+    def test_sd_svs_paxctld_is_running(self):
+        cmd = ["qvm-run", "-p", "sd-svs-disp",
+               "/usr/sbin/service paxctld status"]
+        p = subprocess.check_output(cmd)
+        self.assertTrue("active (running)".encode() in p)
```

Comment:
:+1: on checking the service in the config states. Although we've only been bitten by PaX flags on `sd-svs-disp` so far, we in fact what the paxctld service running in other VMs, as well (see #231). Will adjust the test to run against other machines, as well.